openquake/hazardlib/tests/gsim/data/F19

```diff
@@ -0,0 +1 @@
+
```

Comment:
Why do you have an empty file here?

Reply:
I made this file "F19" incorrectly instead of creating a folder and could not delete it. Please help me in this step to add the GSIM to the software library and run it, what should I do?